app/views/getready.html.erb

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```

Comment:
This file should be renamed to _get_ready.html.erb (because Ruby programmers love snake_case)